setup.py

```diff
@@ -50,6 +50,7 @@ def package_files(directory, strip_leading):
           'typing_extensions',
           'pyfiglet',
           'psutil',
+          'imgaug'
```

Comment:
the imgaug library break the CSIC and the opencv camera on the Jetson nano.  So we should probably add this to the pi for now, but leave it off the nano.

Reply:
I'll change this.